mathesar/api/serializers/columns.py

```diff
@@ -6,7 +6,7 @@
 
 class InputValueField(serializers.CharField):
     """
-    Takes in an arbitrary value. Use to emulate our column and record creation and
+    Takes in an arbitrary value. Use to emulate our column creation and
```

Comment:
The intention with the original comment was to point out that record creation endpoint took in arbitrary values (un-validated and un-processed `request.data`), and this field replicates that behavior but in a serializer. I'm realizing now that this isn't very clear - maybe we can just remove the second sentence?

Reply:
I think we can change the comment at our leisure in a later PR; I'm going to go ahead and merge so we can base work off of this for now.

Reply:
I'm fixing this in the next PR.